components/Seperator.tsx

```diff
@@ -0,0 +1,6 @@
+import { Separator as Seperate } from "@shadcn/ui/separator";
```

Comment:
I would import as is and export our component as Divider. :)

Reply:
Done.
Please, Let me know if there's any prop we would need. 
Now, you can merge to main. ;)